src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public String getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
```

Comment:
let's create a constant for 100
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery